### PR TITLE
Expand OCR test time

### PR DESCRIPTION
**Pull request by kalverra**
Quick fix for consistency issues. Talking to @jleeh about getting a blockchain method to subscribe to new blocks, to make this cleaner and more consistent than the basic check-sleep loop.

### CODE REVIEW

**Review by tateexon — APPROVED** [review]
Looks good to me

File: suite/contracts/contracts_ocr_test.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ var _ = Describe("OCR Feed", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 
 			// Wait for a round
-			for i := 0; i < 10; i++ {
+			for i := 0; i < 60; i++ {
```
Comment (jleeh) [inline suite/contracts/contracts_ocr_test.go:118]: Use the retry lib that Sergey brought in, like here:
https://github.com/smartcontractkit/integrations-framework/blob/main/suite/contracts/contracts_runlog_test.go#L101

**Review by jleeh — CHANGES_REQUESTED** [review]

**Comment by kalverra** [comment]
# Improved OCR Test
Improved this to utilize the new block header subscription to poll only for new rounds, and included a value change check. The previous test time was ~60-70 seconds, this one includes another check, and consistently runs in ~70-75 seconds

File: suite/contracts/contracts_ocr_test.go [hunk 2]
```diff
@@ -101,42 +101,47 @@ var _ = Describe("OCR Feed", func() {
 					P2PBootstrapPeers:  []client.Chainlink{bootstrapNode},
 					KeyBundleID:        nodeOCRKeyId,
 					TransmitterAddress: nodeTransmitterAddress,
-					ObservationSource:  client.ObservationSourceSpec(adapter.ClusterURL() + "/five"),
+					ObservationSource:  client.ObservationSourceSpec(fmt.Sprintf("%s/variable", adapter.ClusterURL())),
```
Comment (jleeh) [inline suite/contracts/contracts_ocr_test.go:104]: I'd add this endpoint within the adapter interface so we don't need to do this string formatting each time.

**Review by jleeh — CHANGES_REQUESTED** [review]

**Review by jleeh — APPROVED** [review]

**Pull request merged**